community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/ReturnAcceptanceTest.scala

```diff
@@ -313,7 +313,7 @@ order by a.age""").toList)
   }
 
   test("renaming in multiple steps should still work") {
-    val result = executeWithRulePlanner(
+    val result = executeWithCostPlannerOnly(
```

Comment:
The test does a lot of weird things with node ids, that makes the results between cost and rule not to be comparable